packages/SwingSet/src/controller/initializeSwingset.js

```diff
@@ -459,6 +477,15 @@ export async function initializeSwingset(
     return getBundle(desc, mode, nameToBundle)
       .then(addBundleID)
       .then(bundleWithID => {
+        const emptyBundleSize = 11320;
```

Comment:
What is the signifance of `11320`? It has a "magic number" vibe.

Reply:
@warner reports that it's the size of an empty bundle. 

Yeah I'll find a better place this constant

Reply:
Second thought, this was for debugging and is obsolete with https://github.com/Agoric/agoric-sdk/pull/7076. It's also a little squishy relying on this empirical value and it's an odd place for this check.

Conferred with @dckc to take it out instead.